This file is part of MXE.
See index.html for further information.

From a1d0ed70e2c94d89d0b598eacca6ac79b8cd5508 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Sun, 2 Sep 2012 00:55:28 +0200
Subject: [PATCH] include gpg-error with gcrypt for static linking
 (mxe-specific)


diff --git a/configure.ac b/configure.ac
index 563fb04..16caa88 100644
--- a/configure.ac
+++ b/configure.ac
@@ -124,6 +124,7 @@ if test "$ac_cv_libgcrypt" = "yes"; then
   AC_DEFINE(LIBSSH2_LIBGCRYPT, 1, [Use libgcrypt])
   LIBSREQUIRED= # libgcrypt doesn't provide a .pc file. sad face.
   LIBS="$LIBS -lgcrypt"
+  LIBS="$LIBS `${prefix}/bin/gpg-error-config --libs`"
   found_crypto=libgcrypt
 fi
 AM_CONDITIONAL(LIBGCRYPT, test "$ac_cv_libgcrypt" = "yes")
-- 
2.1.0

